main.py

```diff
@@ -2,9 +1,41 @@
+import json
+from tkinter.font import names
 
-from db.models import Race, Skill, Player, Guild
+import init_django_orm  # noqa: F401
+import json
+from db.models import Guild, Race, Skill, Player
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as f:
+        players_data = json.load(f)
+    for player_data in players_data:
+        race,_ = Race.objects.get_or_create(name=player_data["race"]["name"],
+                                            defaults = {"description": player_data["race"].get("description", "")}
+        )
+
+
+        guild = None
+        if "guild" in player_data and player_data["guild"]:
+            guild,_ = Guild.objects.get_or_create(
+                name = player_data["guild"]["name"],
+                defaults = {"description": player_data["guild"].get("description", "")}
+            )
+
+        skills = []
+        for skill_data in player_data.get("skills", []):
+            skill,_ = Skill.objects.get_or_create(name = skill_data["name"],defaults = {"bonus": skill_data["bonus"], "race": race})
+            skills.append(skill)
+
+        Player.objects.get_or_create(nickname=player_data["nickname"],defaults={
+            "email": player_data["email"],
+            "bio": player_data["bio"],
+            "race": race,
+            "guild": guild,
+            "create_at": player_data.get("create_at")
```

Comment:
There is a typo in the field name `create_at`. It should be `created_at` to match the field defined in the `Player` model.